audio/out/ao_pipewire.c

```diff
@@ -227,7 +239,7 @@ static void on_param_changed(void *userdata, uint32_t id, const struct spa_pod *
     if (param == NULL || id != SPA_PARAM_Format)
         return;
 
-    int buffer_size = ao->device_buffer * af_fmt_to_bytes(ao->format) * ao->channels.num;
+    int buffer_size = ao->device_buffer * ao->sstride;
```

Comment:
Needs some explanation in the commit message:
`ao->sstride` is alrady initialized to the same value in `init()` but in addition it can also handle planar formats.